lib/SILOptimizer/Mandatory/Differentiation.cpp

```diff
@@ -3910,21 +3912,22 @@ class AdjointEmitter final : public SILInstructionVisitor<AdjointEmitter> {
         remapType(pullback->getType()).castTo<SILFunctionType>();
 
     // Get the seed (i.e. adjoint value of the original result).
-    SILValue seed;
+    ValueWithCleanup seed;
     if (origResult->getType().isObject()) {
       // If original result is a `tuple_extract`, materialize adjoint value of
       // `ai` and extract the corresponding element adjoint value.
       if (auto *tupleExtract = dyn_cast<TupleExtractInst>(origResult)) {
         auto adjointTuple = materializeAdjoint(takeAdjointValue(ai), loc);
-        seed = builder.emitTupleExtract(loc, adjointTuple,
-                                        tupleExtract->getFieldNo());
+        auto seedVal = builder.emitTupleExtract(loc, adjointTuple,
+                                                tupleExtract->getFieldNo());
```

Comment:
note: wrapping style in `Differentiable.cpp` is not consistent, I believe `clang-format` prefers the following:
```cpp
auto seedVal = builder.emitTupleExtract(
    loc, adjointTuple, tupleExtract->getFieldNo());
```

Non-priority aside: I think it'd be nice to run `clang-format` on `Differentiation.cpp` (and accept minor undesirable format decisions) - it would save me minutes per day from thinking about formatting.

Reply:
How's this relevant to this patch? This is not even changing the original formatting. 

I'm not a fan of the auto-formatted code style.

Reply:
Just a side-comment, not relevant to the patch.
Acknowledged, no auto-formatting is fine with me.